taxi/forms.py

```diff
@@ -0,0 +1,48 @@
+from django.contrib.auth import get_user_model
+from django.contrib.auth.forms import UserCreationForm
+from django.core.exceptions import ValidationError
+
+from taxi.models import Driver, Car
+from django import forms
+
+
+class DriverCreationForm(UserCreationForm):
+    class Meta(UserCreationForm.Meta):
```

Comment:
this should have license number validation as well. Also think how to apply inheritance so you define your forms more efficiently